.github/workflows/android_build_ubdiag_upload.yml

```diff
@@ -0,0 +1,108 @@
+name: Build and Upload to UBDiag
+
+on:
+  push:
+  workflow_call:
+    inputs:
+      projectKey:
+        type: string
+        required: true
+      app:
+        type: string
+        required: true
+      flavor:
+        type: string
+        required: true
+      appModuleDirectory:
+        type: string
+        required: false
+        default: './app'
+    secrets:
+      ANDROID_JENKINS_PAT:
+        required: true
+      UB_ARTIFACTORY_URL_ANDROID:
+        required: true
+      UB_ARTIFACTORY_USERNAME:
+        required: true
+      UB_ARTIFACTORY_PASSWORD:
+        required: true
+      UBIQUE_POEDITOR_API_KEY:
+        required: true
+      UBDIAG_UPLOAD_URL:
+        required: true
+      ADDITIONAL_GRADLE_PROPS:
```

Comment:
Warum ist das im secrets?

Reply:
Ich hatte es erst in den inputs, aber der calling Workflow hatte im `with` irgendwie keinen Zugriff auf die Environment Secrets. Siehe hier: https://github.com/UbiqueInnovation/sac-android/actions/runs/1751870972/workflow